cmd/test/cmd.go

```diff
@@ -0,0 +1,102 @@
+package main
+
+import (
+	"os"
+
+	"github.com/spf13/cobra"
+
+	"github.com/PermissionlessGames/degen-casino/bindings/Tests/TestableDegenGambit"
+	"github.com/PermissionlessGames/degen-casino/version"
+)
+
+func CreateRootCommand() *cobra.Command {
+	// rootCmd represents the base command when called without any subcommands
+	rootCmd := &cobra.Command{
+		Use:   "test",
```

Comment:
`test` is so misleading. Can you add this in `technician`?

Reply:
Yes, I can move this into technician instead of having a seperate cmd system